<?xml version="1.0" encoding="UTF-8" ?>
<launch>

    <node name="turning_test" pkg="rr_iarrc" type="turning_test" output="screen">
        <param name="turn_speed" type="double" value="0.6"/>
        <param name="left_turn_steering" type="double" value="-0.20"/>
        <param name="right_turn_steering" type="double" value="0.20"/>
        <param name="left_offset" type="double" value="0.0"/>
        <param name="right_offset" type="double" value="0.0"/>
       	<param name="forward_timeout" type="double" value="10.0"/>
        <param name="turn_direction" type="string" value="forward"/>
    </node>

</launch>